unix/x0vncserver/x0vncserver.cxx

```diff
@@ -174,6 +177,21 @@ class XDesktop : public SDesktop, public TXGlobalEventHandler
 #ifdef HAVE_XDAMAGE
     }
 #endif
+
+#ifdef HAVE_XFIXES
+    int xfixesErrorBase;
+
+    if (XFixesQueryExtension(dpy, &xfixesEventBase, &xfixesErrorBase)) {
+      if (not haveDamage)
+        TXWindow::setGlobalEventHandler(this);
```

Comment:
It's probably cleaner to always set the event handler and handle it there. Avoids interdependencies like this.